### PR TITLE
refactor: Hook `useWalletBalance`

### PR DESCRIPTION
**Pull request by aleksaToljic**
# Pull Request Template

Feel free to remove any section (except the checklist) from this template.
But please try to fill in as much of this as possible.

## Description

Please include a summary of the proposed changes, including relevant motivation and context.

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a configuration update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide easily reproducible instructions.
List any relevant details for your test configuration.

## Checklist:

- [ ] I have performed a self-review of this code and spelling
- [ ] I have added comments in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Additional information

Include any additional information about these changes here.

For example, if you are making a visual change, please include BEFORE (if any) and AFTER screenshots.
A visual change could be just reordering some components on a page.
Any visual changes in the UI MUST provide these screenshots.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
     "styled-components": "^5.3.5"
   },
   "devDependencies": {
+    "@polkadot/util-crypto": "^10.1.11",
+    "@polkadot/util": "^10.1.11",
```
Comment (aleksaToljic) [inline package.json:41]: added because it was required as `"peerDependency"`

File: packages/core/src/hooks/useWalletBalance.tsx [hunk 2]
```diff
@@ -77,14 +63,33 @@ export const useWalletBalance = ({
   currentAccount,
   currentChainId,
 }: Props = {}): TokenBalanceData[] => {
-  const balanceReturned: TokenBalanceData[] = []
+  const xcmSdk = init()
+  const [balances, setBalances] = useState<TokenBalanceData[]>()
```
Comment (aleksaToljic) [inline packages/core/src/hooks/useWalletBalance.tsx]: hm, I forgot to add empty array here. Will do in next commit.

File: package.json [hunk 3]
```diff
@@ -37,6 +37,10 @@
...
+    "@polkadot/wasm-crypto": "^6.3.1",
```
Comment (aleksaToljic) [inline package.json:42]: was needed as well for `useWalletBalances` to pass the unit test.

File: package.json [hunk 4]
```diff
@@ -37,6 +37,10 @@
...
+    "@polkadot/x-randomvalues": "^10.1.9",
```
Comment (aleksaToljic) [inline package.json:43]: was a peer dependencies to `@polkadot/wasm-crypto`

**Review by Kristina2103 — DISMISSED** [review]

**Review by vculum-bloxico — DISMISSED** [review]

**Pull request merged**